truck_signs_designs/settings/base.py

```diff
@@ -154,4 +156,6 @@
 # EMAIL_HOST_USER = os.getenv("EMAIL_HOST_USER")
 # EMAIL_HOST_PASSWORD = os.getenv("EMAIL_HOST_PASSWORD")
 
-# Activate Django-Heroku.
\ No newline at end of file
+# Activate Django-Heroku.
+
+ALLOWED_HOSTS= env("ALLOWED_HOSTS")
```

Comment:
If you have more than 1 allowed host, you have to convert it everytime as a list:
```suggestion
ALLOWED_HOSTS = env.list("ALLOWED_HOSTS")
```